imports/ui/Editor.jsx

```diff
@@ -60,18 +61,28 @@ export class Editor extends Component {
     this.editor.on('change', () => {
       this.props.change(this.editor.getData(), this.editor.editable().getText())
     })
+
+    Meteor.call('settings.find', (e, obj) => {
```

Comment:
Change this! Instead of returning a whole setting doc to the client, just return image properties, e.g. 'settings.getImageUploadSettings'.
Also, search  everywhere for where settings.find is called - should probably never have to call it, and should probably just delete the method.